All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+-|-
+ **11.1**        | Introduce EIP 2046 in a future AllCoreDevs call to be considered as an EFI.
+**11.2**        | Discuss an emergency communications checklist on the next EIPIP call.
+**11.3**        | Pooja to add Jaye Harill to the comms group in the Cat Herders
+**11.4**        | BFollow up with lightclient on his new validator and the rules.
+**11.5**        | Read [this link](https://www.rfc-editor.org/about/independent/#:~:text=To%20post%20an%20Internet%2DDraft,%40rfc%2Deditor.org.) about independent submissions as part of the RFC before the next meeting.
+**11.6**       | James to write the precompile discussion item into a PR.
+**11.7**       | James to work on doing a document or a PR for the statuses to EIP 1
+**11.8**       | Look at the Cat Herders to investigate the project board for the EIPIP repo.
+**11.9**       | Figure out who has access to give Edson write access to the Cat Herders repo.
+
+
+
+**James Hancock**
+Let's start. I'm James Hancock. This is EIPIP meeting 11, with the [agenda on the Cat Herders GitHub.](https://github.com/ethereum-cat-herders/EIPIP/issues/21) We are small today, so we're going to review and go over stuff and perhaps have a short meeting. But thank you all who came. If there's something particular you all want to go over, since we're small enough, I think we can be flexible about what to talk about first, if there's anything that comes to mind.
+
+## 7. Using the Projects feature of Github to organize talking points for future EIPIP meetings.
+
+**Pooja Ranjan**
+So yeah, we can start with the idea of the project board. That is something new and interesting that we might need brainstorming over there. So just introduce why and how you thought of it and where you see this going.
+
+**James Hancock**
+Okay. You want to talk about your thoughts on this?
+
+**Edson Ayllon**
+Yeah, so I was watching you use it for EFI. And it looks like Trello. Sometimes, there's things on the agenda, which we don't get to. As more and more items are added to the agenda, I think it'll add bloat to future agendas, and we might lose items. So I think it's a good way to organize future talking points that we table to future meetings. Or sometimes they have agenda items that - say we want to focus on decision making, but we're also introducing stuff around on-boarding. We still have ideas that we want to keep, but not necessarily address at this moment. And a way to organize them could be using the project's feature and GitHub. So it's more of a way to organize the topics from these meetings.
+
+**Pooja Ranjan**
+Yeah, I kind of like the idea, too. Sometimes back, we have used it for creating the post-mortem for constant in poll and the Cat Herders itself. I'm gonna share the link in the chat here. So yeah, this is a good way to keep track of whatever agenda that we are trying to discuss here, and of any decisions we make, and how we are going to implement that. It would be a good way to keep track of it.
```

Comment:
"Sometimes back, we have used it for creating the post-mortem for constant in poll and the Cat Herders itself. I'm gonna share the link in the chat here." -> "Sometimes back, we have used it for creating the post-mortem for Constantinople in the Cat Herders repo itself. I'm gonna share the [link in the chat](https://github.com/orgs/ethereum-cat-herders/projects/1) here."